lib/src/mirror_utils/with_mirrors.dart

```diff
@@ -38,13 +38,23 @@ FunctionData loadFunctionData(Function fn) {
     // Get parameter name
     String name = MirrorSystem.getName(parameterMirror.simpleName);
 
+    Iterable<T> getAnnotations<T>() =>
+        parameterMirror.metadata.map((e) => e.reflectee).whereType<T>();
+
+    // If present, get name annotation and localized names
+    Map<Locale, String>? nameLocales;
+    Iterable<Name> nameAnnotations = getAnnotations<Name>();
+
+    if (nameAnnotations.isNotEmpty) {
```

Comment:
Should have a check to ensure there is at most one `@Name` annotation like the other annotations.